### PR TITLE
Adding myself as LT and Maintainer for Cisco platforms

### PR DESCRIPTION
**Pull request by edolnx**
We added Cisco to Tier 1 support, and it's already available. Just filling out the paperwork.

### CODE REVIEW

**Comment by coderanger** [comment]
Any reason to not just make a single component? Also we need a majority of existing LTs so y'all should formally +1 this.

**Comment by edolnx** [comment]
We're hoping that Cisco will take over these platforms in the long run, and it would be separate teams/people within their organization. Hence why we broke it out into two.

**Comment by coderanger** [comment]
Fair nuff' :)

File: MAINTAINERS.toml [hunk 1]
```diff
@@ -161,6 +161,26 @@ The specific components of Chef related to a given platform - including (but not
...
+
+        lieutenant = "cperry"
+
+        maintainers = [
+          "cperry"
+        ]
+
+      [Org.Components.Subsystems.CiscoIOSXR]
+        title = "Cisco IOS XR"
+        team = "client-isoxr"
```
Comment (coderanger) [inline MAINTAINERS.toml]: Typo?

Comment (edolnx) [inline MAINTAINERS.toml]: Yes it is, pushing new update.

**Comment by coderanger** [comment]
Ping @thommay @sethvargo @jonlives @ranjib @btm @lamont-granquist @jtimberman @martinisoft @joemiller 

We need 5 of you to vote for it I think.

**Comment by martinisoft** [comment]
:+1:

**Comment by lamont-granquist** [comment]
:+1:

**Comment by btm** [comment]
Run the rake task to update the MD?

:+1:

**Comment by thommay** [comment]
:+1:

**Comment by jonlives** [comment]
:-1:

**Comment by coderanger** [comment]
The LTs have spoken!

**Comment by coderanger** [comment]
@btm I'll handle re-generating the markdown.

**Pull request merged**